internal/pkg/service/common/task/node_test.go

```diff
@@ -38,11 +45,13 @@ func TestSuccessfulTask(t *testing.T) {
 	etcdNamespace := "unit-" + t.Name() + "-" + idgenerator.Random(8)
 	client := etcdhelper.ClientForTestWithNamespace(t, etcdNamespace)
 	logs := ioutil.NewAtomicWriter()
+	tel := telemetry.NewForTest(t)
 
 	// Create nodes
-	node1, _ := createNode(t, etcdNamespace, logs, "node1")
-	node2, _ := createNode(t, etcdNamespace, logs, "node2")
+	node1, _ := createNode(t, etcdNamespace, logs, tel, "node1")
+	node2, _ := createNode(t, etcdNamespace, logs, tel, "node2")
 	logs.Truncate()
+	tel.Reset()
```

Comment:
`mocked.Dependencies` obsahuju automaticky `telemetry.ForTest`, tu ale testujeme 2 nodes, takze tu telemetriu tam nastavime externe a rovnaku. Inak by sme mali 2 instancie, a museli by sme to 2x skontrolovat.